### PR TITLE
Add missing doc aliases

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

File: gdk/src/device.rs [hunk 1]
```diff
@@ -26,6 +26,7 @@ impl Device {
 
     #[doc(alias = "gdk_device_get_history")]
     #[doc(alias = "get_history")]
+    #[doc(alias = "gdk_device_free_history")]
```
Comment (bilelmoussaoui) [inline gdk/src/device.rs:29]:
```suggestion
```
not needed

File: gtk/src/drag_context.rs [hunk 2]
```diff
@@ -24,6 +24,7 @@ pub trait DragContextExtManual: 'static {
     fn drag_set_icon_name(&self, icon_name: &str, hot_x: i32, hot_y: i32);
 
     #[doc(alias = "gtk_drag_set_icon_pixubf")]
+    #[doc(alias = "gtk_drag_set_icon_pixbuf")]
```
Comment (bilelmoussaoui) [inline gtk/src/drag_context.rs:27]: hmm i can't tell the difference with the existing doc alias here?

File: gtk/src/rt.rs [hunk 3]
```diff
@@ -103,6 +103,7 @@ pub unsafe fn set_initialized() {
 /// so will not cause the program to terminate if GTK could not be initialized.
 /// Instead, an Ok is returned if the windowing system was successfully
 /// initialized otherwise an Err is returned.
+#[doc(alias = "gtk_init_check")]
```
Comment (bilelmoussaoui) [inline gtk/src/rt.rs:106]: not needed, maybe make it  `gtk_init` as it's the right one to use?

File: gtk/src/drag_context.rs (hunk 2 above)
Comment (GuillaumeGomez) [reply gtk/src/drag_context.rs:27]: I can't either, I guess it's a bug. :)

File: gtk/src/rt.rs (hunk 3 above)
Comment (GuillaumeGomez) [reply gtk/src/rt.rs:106]: I'll make a replacement for it.

File: gtk/src/drag_context.rs (hunk 2 above)
Comment (GuillaumeGomez) [reply gtk/src/drag_context.rs:27]: Oh I see it now! The `u` and `b` are not in the right place. That's pretty funny. :)

**Comment by GuillaumeGomez** [comment]
Updated!

File: gtk/src/drag_context.rs (hunk 2 above)
Comment (bilelmoussaoui) [reply gtk/src/drag_context.rs:26]: lol

**Review by bilelmoussaoui — APPROVED** [review]

**Pull request merged**